### ISSUE TITLE
Stack of errors for long error messages

### ISSUE THREAD

**Issue by shimarulin** [mikaelmello/inquire#21]
**Describe the bug**
If the error message is longer than the available terminal width, the error messages start to appear one below the other when any key is pressed.

**To Reproduce**
1. Add long message to validator
2. Set terminal window less then validation message length
3. Get validation error
4. Press left arrow key or backspace or continue typing

**Expected behavior**
The error message keeps its position on the screen.

**Screenshots**
![image](https://user-images.githubusercontent.com/5099498/130335452-fb70f72f-2dd5-4433-af88-d575b34cd4f7.png)

**Desktop**
 - OS: ArchLinux
 - Version: Latest

**Additional context**
Can be related to #15

**Comment by mikaelmello** [comment]
Thanks for reporting, I'm taking a look at fixing #15 at the moment, working with terminal sizes will be the next step!

**Comment by mikaelmello** [comment]
There was a trade-off to be made.

The easiest fix was to disable line wrapping, leaving the "nice-UI" responsibility in these cases to the lib user. However, since I don't provide any API for the user to properly break their own lines based on the terminal size, I discarded this idea.

The other fix was to calculate how many lines the rendered text actually occupied. Besides counting the \n characters in all of the written text, lines that are too long will be automatically splitted by the terminal. We then calculate the current terminal width to check how many lines the "current line" actually occupied.

The first option for this fix was to get the terminal size and calculate the line count on every re-render. But this didn't magically fix some problems when resizing the terminal either way. Thus, I have decided to just retrieve the terminal size at the prompt start-up and assume it won't be changed until it is finished.
